apps/api/services/zpe/worker_auth.py

```
@@ -0,0 +1,102 @@
+    ttl_seconds: int
+
+
+class WorkerTokenStore:
+    def __init__(self, redis: Optional[Redis] = None) -> None:
+        self.redis = redis or get_redis_connection()
+
+    def create_token(self, worker_id: str, *, label: Optional[str] = None) -> WorkerToken:
+        settings = get_zpe_settings()
+        ttl = int(settings.worker_token_ttl_seconds)
+        if ttl <= 0:
+            raise ValueError("worker_token_ttl_seconds must be >= 1")
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)